src/fights.ts

```diff
@@ -529,11 +530,18 @@ function bestFairy() {
 class FreeFight {
   available: () => number | boolean;
   run: () => void;
+  tentacle: () => boolean;
```

Comment:
I don't see the utility in making this a function rather than a boolean--it seems like a property that does not change.

Reply:
Yeah, I am still a bit hazy on when to use one or tother, so I was just following how available and run were done. Changing